### DISCUSSION TITLE
신고 엔티티 관련

### DISCUSSION THREAD

**Discussion by hodin030** [cca-ffodregamdi/running-hi-back#246] [Questions]
현재 게시글 신고 엔티티에 신고된 post를 joincolumn하고 있지 않고 삭제여부 필드만 있는데 그렇게 변경한 이유를 알 수 있을까요?
피신고자 정보를 알려면 해당 post 정보를 가져와서 작성자 컬럼을 조회해야할 것 같아서 물어봐용

**Comment by numerical43** [comment]
변경한건 아니고 처음에 post entity가 만들어지기 전이라서 임시로 추가해야한다는 주석만 달아놨었습니다!
post entity 만들어지고 joincolumn 추가하려고 할 때 효댕씨 합류 소식을 듣고, report domain을 더이상 안건드려서 그렇게 되어버렸습니다😂
그 부분까지는 했어야했는데, 혼란드려서 죄송해요!!

**Reply by hodin030** [reply]
앗 나 이거 올린지 모르고 있었네 지금봤다 답변 감사합니당
